model.py

```diff
@@ -90,7 +90,7 @@ def generate_poll_summary_with_link(self):
     def render_text(self):
         header = [util.make_html_bold_first_line(self.title)]
         body = [option.render_text() for option in self.options]
-        footer = [u'\U0001f465 ' + self.generate_respondents_summary()]
+        footer = [util.emoji_people_unicode() + self.generate_respondents_summary()]
```

Comment:
I'd prefer if the space were left out of the util method. This has also resulted in two consecutive spaces in `render_text`.